Resources/Prototypes/_Impstation/Entities/Mobs/NPCs/miscellaneous.yml

```diff
@@ -168,3 +168,44 @@
     interactSuccessSound:
       path: /Audio/_Impstation/Animals/mwah.ogg
     interactSuccessSpawn: EffectHearts
+
+- type: entity
+  parent: PumpkinLantern
+  id: PumpkinLanternReallyLarge
+  suffix: ReallyLarge
+  components:
+    - type: Sprite
+      scale: 4, 4
+    - type: GhostTakeoverAvailable
+    - type: GhostRole
+      allowMovement: false
+      allowSpeech: true
+      makeSentient: true
+      name: jack
+      description: this holiday is for YOU and everyone better act like it
+      raffle:
+        settings: default
+
+- type: entity
+  parent: MobUraniumCrab
+  id: MobRay
+  description: An ore crab. It has a sticker on it's carapace reading "PLEASE HANDLE WITH CARE".
+  components:
+  - type: Sprite
+    scale: 3, 3
+  - type: PointLight
+    radius: 3
+    energy: 4
+    color: "#06DF24"
+
+- type: entity
+  id: MobSpawnRay
```

Comment:
similarly, please make a proper spawner for this. you can see the spawners in `Resources/Prototypes/_Impstation/Entities/Markers/Spawners/mobs.yml`

Reply:
done and added some stuff for petting them